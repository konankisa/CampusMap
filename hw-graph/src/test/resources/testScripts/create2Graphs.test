# Creates 2 empty graphs

# Create a graph
CreateGraph graph1

# Create another graph
CreateGraph graph2