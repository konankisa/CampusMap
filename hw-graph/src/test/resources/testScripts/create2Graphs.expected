# Creates 2 empty graphs

# Create a graph
created graph graph1

# Create another graph
created graph graph2